src/renderer/helpers/rx/fromPromise.ts

```diff
@@ -6,7 +6,7 @@ import { observableState } from '../stateHelper'
 export const fromPromise$ = <T, V>(
   factory: (next: V) => Promise<T>,
   defaultValue: T,
-  onError = (_: Error) => defaultValue
+  onError: (_: Error) => T = (_) => defaultValue
```

Comment:
^ Not needed to fix type issues in `tests` - just to be more explicit in type annotations